rust/src/db.rs

```diff
@@ -50,3 +52,50 @@ fn get_db() -> Result<MutexGuard<'static, SqlitePool>> {
         .lock()
         .map_err(|_| anyhow!("cannot acquire DB lock"))
 }
+
+pub async fn load_cfds(conn: &mut SqliteConnection) -> Result<Vec<Cfd>> {
```

Comment:
❓ do we want to keep the sql queries here? I started adding them to the cfd file, but this is again growing quite big, so I wouldn't mind putting the queries somewhere else. However, adding it to the db file will bloat the db again.

Should we create a dedicated cfd module with the following structure?

```
open.rs
settles.rs
load_cfds.rs
insert_cfd.rs
```

Reply:
see bf6d650c65ed8fc08e2f0aca252b13f1e89cb090